src/components/Questionnaire/CodingEditor.tsx

```diff
@@ -111,6 +111,31 @@ export function CodingEditor({
 
       <CardContent className="space-y-4">
         <div>
+
+          <Label className="block mb-1">System</Label>
+          <Select
```

Comment:
This doesn't look right? Now you have the select outside FormField. Let's make changes to ones inside FormField rather than moving it out.

Same thing for code.